protocols/luci-proto-wireguard/htdocs/luci-static/resources/protocol/wireguard.js

```diff
@@ -88,14 +95,33 @@ return network.registerProtocol('wireguard', {
 		o.validate = validateBase64;
 		o.rmempty = false;
 
+		var sections = uci.sections('network');
+		var serverName = this.getIfname();
+		var server = findSection(sections, serverName);
+
+		o = s.taboption('general', form.Value, 'public_key', _('Public Key'), _('Base64-encoded public key of this interface for sharing.'));
+		o.rmempty = false;
```

Comment:
Just for understanding, is this new option supposed to be editable by the user or is this just an informational display thing? In the latter case consider either using [`form.DummyValue`](https://openwrt.github.io/luci/jsapi/LuCI.form.DummyValue.html) or [`o.readonly = true`](https://openwrt.github.io/luci/jsapi/LuCI.form.Value.html#readonly)

Reply:
Also no clear way to update the `public` key HTML when using `form.DummyValue`, and one generates a new key(pair). I thought I came up with a reason why not to `o.write` but seem to have forgotten what that was.